HZETRN/btrn.py

```diff
@@ -0,0 +1,377 @@
+#########################################################
+# btrn.py
+# Backscatter Transport and Response
+#########################################################
+
+import subprocess
+import argparse
+import string
+import os
+import pprint
+
+try:
+   from pyne import material
+   from pyne.material import Material, MaterialLibrary
+except:
+    raise ImportError("The PyNE dependencies could not be imported.")    
+
+try:
+    from pyne import dagmc
+except:
+    raise ImportError("The dagmc dependency could not be imported.")    
+
+from pyne import data
+from pyne import nucname
+import numpy as np
+import hzetrn as one_d_tool
+import tag_utils
+import logging
+
+
+###############################
+# globals
+config_log  = 'config.log'
+# 'spatial_dir' is a subdir for storing input files as they are created
+spatial_dir = 'spatial'
+
+"""
+    Read the coordinates of the Reference Point from a file.
+    filename = name of file containing triplet
+        If filename = '' the reference point is set
+        to the origin
+"""
+def load_ray_start(filename):
+    # Set the Reference Point.
+    ray_start = []
+    if not filename:
+        return [0.0, 0.0, 0.0]
+    else:
+        with open(filename) as f:
```

Comment:
with open(filename) throws an exception and exits if the file is not found:
IOError: [Errno 2] No such file or directory: 'doesnotexist'
It seems to me this is exactly how I would want it handled anyway.    Again, this code is going away, though the same construction is used in reading the list of directions.